cmd/auth-provider-gcp/credentialconfig/README.md

```diff
@@ -0,0 +1 @@
+Temporarily copied from a PR intended to be merged into into kubernetes/kubernetes (https://github.com/kubernetes/kubernetes/pull/95775) - please do not modify this code!
```

Comment:
Here too.

Reply:
I added an expanded note and some removal criteria; the exact import path of the replacement is a bit of an unknown since the PR this came from still needs to be modified/updated, so I didn't feel comfortable putting a new path down yet.

Reply:
Resolving since the issue has been made (#173) - the import path can be mentioned there as https://github.com/kubernetes/kubernetes/pull/95775 gets closer to merging.